pkg/network/ovn_kubernetes.go

```diff
@@ -623,52 +643,60 @@ func bootstrapOVN(conf *operv1.Network, kubeClient crclient.Client) (*bootstrap.
 	controlPlaneReplicaCount, _ := strconv.Atoi(rcD.ControlPlane.Replicas)
 
 	var heartBeat int
+	var hypershift string = os.Getenv("HYPERSHIFT")
 
-	err = wait.PollImmediate(OVN_MASTER_DISCOVERY_POLL*time.Second, time.Duration(OVN_MASTER_DISCOVERY_TIMEOUT)*time.Second, func() (bool, error) {
-		matchingLabels := &crclient.MatchingLabels{"node-role.kubernetes.io/master": ""}
-		if err := kubeClient.List(context.TODO(), masterNodeList, matchingLabels); err != nil {
-			return false, err
-		}
-		if len(masterNodeList.Items) != 0 && controlPlaneReplicaCount == len(masterNodeList.Items) {
-			return true, nil
-		}
+	ovnMasterIPs := make([]string, controlPlaneReplicaCount)
 
-		heartBeat++
-		if heartBeat%3 == 0 {
-			klog.V(2).Infof("Waiting to complete OVN bootstrap: found (%d) master nodes out of (%d) expected: timing out in %d seconds",
-				len(masterNodeList.Items), controlPlaneReplicaCount, OVN_MASTER_DISCOVERY_TIMEOUT-OVN_MASTER_DISCOVERY_POLL*heartBeat)
-		}
-		return false, nil
-	})
-	if wait.ErrWaitTimeout == err {
-		klog.Warningf("Timeout exceeded while bootstraping OVN, expected amount of control plane nodes (%v) do not match found (%v): %s, continuing deployment with found replicas", controlPlaneReplicaCount, len(masterNodeList.Items))
-		// On certain types of cluster this condition will never be met (assisted installer, for example)
-		// As to not hold the reconciliation loop for too long on such clusters: dynamically modify the timeout
-		// to a shorter and shorter value. Never reach 0 however as that will result in a `PollInfinity`.
-		// Right now we'll do:
-		// - First reconciliation 250 second timeout
-		// - Second reconciliation 130 second timeout
-		// - >= Third reconciliation 10 second timeout
-		if OVN_MASTER_DISCOVERY_TIMEOUT-OVN_MASTER_DISCOVERY_BACKOFF > 0 {
-			OVN_MASTER_DISCOVERY_TIMEOUT = OVN_MASTER_DISCOVERY_TIMEOUT - OVN_MASTER_DISCOVERY_BACKOFF
+	if hypershift != "true" {
```

Comment:
Can we pull this out in to another function, something like `getMasterAddresses()`?

Reply:
Done